scripts/load/load_timescaledb.sh

```diff
@@ -53,13 +53,11 @@ cat ${DATA_FILE} | gunzip | $EXE_FILE_NAME \
                                 --in-table-partition-tag=${IN_TABLE_PARTITION_TAG} \
                                 --hash-workers=${HASH_WORKERS} \
                                 --time-partition-index=${TIME_PARTITION_INDEX} \
-                                --partition-on-hostname=${PARTITION_ON_HOSTNAME} \
```

Comment:
What's the reason for modifying this file?

Reply:
done

Reply:
https://github.com/timescale/tsbs/pull/198